src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/energy/EnergyNet.java

```diff
@@ -288,6 +293,10 @@ private int tickAllCapacitors() {
         return supply;
     }
 
+    private boolean isSaturated() {
```

Comment:
Any reason this is private? Could be useful for Addons. Also you should probably add a javadoc.

Reply:
No reason really. It's public now.